provd/tools/tools.go

```diff
@@ -0,0 +1,9 @@
+//go:build tools
```

Comment:
ok, I think another PR will introduce dependabot to get those autoupdated too?

Reply:
Yes, I will create a pr for updating the `dependabot.yaml` :+1:  these do seem to be currently out of its scope